PROJECT

```diff
@@ -92,6 +92,10 @@ resources:
   version: v1alpha1
   kind: MySQLVNetRule
 - group: azure
+  kind: MySQLUser
+  version: v1alpha1
+- group: azure
+  kind: AzureVirtualMachine
```

Comment:
I think this got accidently added again - need to remove ` kind: AzureVirtualMachine`